src/Components/UserProfile.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import "../styles/UserProfile.css"
```

Comment:
@CoreenCooper and @durdonadjalilova I think there is a typo in the filename of this import statement. You have a `userProfile.css` file but check the capitalization in your import statement.

Reply:
Thanks Mike. The relative path was incorrect. It should have been ./styles/UserProfile.css. I corrected it and pushed it. I am wondering why an error wasn't picked up by npm test. It passed all the test locally.

Reply:
Great observation. It's a head scratcher! 🤔 
One possibility is that `npm test` was ignoring that test and only running tests on the changed files. I have experienced myself with the new menu in the terminal. Hitting `a` to run all tests should ensure that every test runs.

Another thing that we saw happening in some of our automation was that in a case like this where there is a typo in the import, the test doesn't run at all. This makes some of our tools think that the test didn't exist. Working on a fix for this but no ETA.

Reply:
Anyway I think the path might still be off here? The name of your file is `userProfile` (not `UserProfile`) so your import statement still can't find it. You can change the file name to `UserProfile` or you can change the import statement to look for `userProfile`. Whatever works as long as they match!

Reply:
You're right I took another look at the address. But as far as "userProfile", I must be over looking it. I can only find the className on line 13 of UserProfile.js with the lowercase "u" .  I tried reading the circleci error, but I can't figure out how to fix it.

![Image of circleci error]
(<img width="1440" alt="Screen Shot 2021-04-04 at 6 52 15 PM" src="https://user-images.githubusercontent.com/38697344/113523962-68659000-9579-11eb-85e3-6a1be4a0d05b.png">)